ts/components/SectionStatusComponent.tsx

```diff
@@ -31,6 +31,8 @@ const styles = StyleSheet.create({
     alignItems: "flex-start",
     alignContent: "center"
   },
+  alignCenter: { alignSelf: "center" },
+  verticalFlipped: { transform: [{ rotateZ: "180deg" }] },
```

Comment:
May we remove this unused style attribute?

Reply:
removed within https://github.com/pagopa/io-app/pull/2578/commits/6c560e71089ada8fc2c58c4b534ec88f3096b15a